packages/ui-extensions/docs/surfaces/point-of-sale/mdxExamples/discount-example/shopify.extension.toml

```diff
@@ -1,5 +1,5 @@
 ...
 name = "Loyalty discount"
-handle = "Loyalty discount"
+handle = "loyalty-discount"
 description = "Add loyalty discount"
```

Comment:
@NathanJolly the guide doesn't specifically mention the discount we're building to be loyalty related. Can we drop loyalty from these three lines and simplify to discount?